ocaml/tests/test_psr.ml

```diff
@@ -0,0 +1,142 @@
+let checkpoint = ref None
+
+let backups = ref None
+
+let default_ps = "default_ps"
```

Comment:
The code uses acronyms in identifiers for modules, types, and values: ps and cp. Looking from the outside, this can be hard to read. I think some more explicit names for modules would help the reader.